apps/i18n/codebridge/en_us.json

```diff
@@ -25,7 +26,9 @@
   "makeSupport": "Make support file",
   "makeValidation": "Make validation file",
   "moveFile": "Move file",
+  "moveFolder": "Move folder",
   "moveFilePrompt": "Please enter your destination folder",
+  "moveFolderPrompt": "Please enter your destination folder",
```

Comment:
I'd checked with Sam and she said she would like to keep it, which is why I'd kept it in place.

Fixed the bug regarding an empty prompt to the root (you _could_ previously move to `/`, but that wasn't very discoverable).

I added a placeholder for `project root` (which is localized) for when there is no input, to give better user feedback.

Reply:
Makes sense! It might be worth checking with Jamila as to what the best terminology to use for `project root` is, I'm not sure what would make the most sense for students. But that could be changed as a follow-up.